tests/test_copydb.py

```diff
@@ -81,10 +81,28 @@ def ir_config_param_test_values(odoodb):
         for key in _get_reset_config_params_key():
             env.cr.execute(
                 """
-            UPDATE ir_config_parameter set value ='test value' where key=%s
+            UPDATE
+                ir_config_parameter
+            SET
+                value='test value'
+            WHERE
+                key=%s
+            RETURNING id
             """,
                 (key,),
             )
+            if not env.cr.fetchall():
+                # Config parameter doesn't exist: create (ex enterprise params)
+                env.cr.execute(
+                    """
+                INSERT INTO
+                    ir_config_parameter
+                    (key, value)
+                VALUES
+                    (%s, 'test value')
+                """,
+                    (key,),
+                )
```

Comment:
nit: with `insert into ... on conflict (key) do update set value=%s` that can be done in one statement.